workshops/request.html

```diff
@@ -4,8 +4,37 @@
 title: "Request a Workshop"
 nocomments: 1
 ---
-<div><p>If you&#39;re interested in hosting a Software Carpentry workshop at your institution, please read our <a href="{{page.root}}/workshops/operations.html">Operations Guide</a> for more information.</p><p>If you are ready to host a workshop, submit this form and someone from the Software Carpentry team will be in touch with you within a week.</p>
-<p><div class="ss-required-asterisk">* Required</div></p></div>
+<div>
+  <p>
+    If would like to host a workshop at your institution,
+    please fill in this form to let us know a bit more about your needs
+    and someone will contact you as soon as possible.
+    For more information on how our workshops operate,
+    please read our <a href="{{page.root}}/workshops/operations.html">Operations Guide</a>.
+  </p>
+  <p>
+    Our instructors are volunteers,
+    and so are not paid for their teaching,
+    but host sites are required to cover their travel and accommodation costs.
+    In addition,
+    the Software Carpentry Foundation
+    <a href="{{page.root}}/scf/workshops.html">charges an administration fee</a>
```

Comment:
And, actually, this is tricky... neither link actually says HOW to pay the fee. Both of them should say how to pay. Honestly, if you asked me tomorrow to pay a workshop fee, I, Katy Huff, would have to send you an email to find out how to do it. It's not clear at all from the website where the fees should be paid and who needs to be informed.